src/librustdoc/html/static/rustdoc.css

```
@@ -1570,7 +1570,8 @@ h4 > .notable-traits {
 		height: 73px;
 	}
 
-	#main {
+	/* This is to prevent the search bar to be under the corresponding section. */
```

The way this comment is worded confuses me. What does 'under the corresponding section' mean?

---

The wording isn't great indeed. What about "the section following it"?

---

Maybe something like

```suggestion
	/* This is to prevent the search bar from being underneath the following section. */
```

then? What's the ID of the following section? It might be good to add that.

---

More like "the element following it". Might be better this way haha.

---

> Might be better this way haha.

Which way is "this way"? ;)

---

Updated! Tell me if the new wording is good for you. :)